web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -544,6 +554,13 @@ export default defineComponent({
             mdi-refresh
           </v-icon>
         </v-btn>
+        <v-snackbar
+          v-model="snackbar"
+          color="success"
+          timeout="3000"
+        >
+            Validation Passed! You can now submit or continue editing.
+        </v-snackbar>
```

Comment:
```suggestion
        >
          Validation Passed! You can now submit or continue editing.
        </v-snackbar>
```

Last one, I swear. The text is over-indented by 2 spaces